go.mod

```diff
@@ -351,3 +350,6 @@ replace github.com/openshift/assisted-service/models => github.com/openshift/ass
 replace github.com/containerd/containerd => github.com/containerd/containerd v1.6.26
 
 replace github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels => github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels v0.0.0-20240404200847-de75746a9505
+
+// This is to force capi back for the older provider version
+replace sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.19.3
```

Comment:
Is there an issue using the new version?

Reply:
In order to not have to leave at previous version, we'll need to update most of the other capi providers to be compatible.  I was not wanting to update them as part of this feature, but if you prefer, I can bump them all up to a newer version that supports that controller-runtime.